Add vSphere mixin (#1198)
grafana/jsonnet-libs: vsphere/README.md

# vSphere Mixin

The vSphere mixin is a set of configurable Grafana dashboards and alerts.

The vSphere mixin contains the following dashboards:

- vSphere cluster
- vSphere overview
- vSphere hosts
- vSphere virtual machines
- vSphere logs

and the following alerts:

- vSphereHostInfoCPUUtilization
- vSphereHostWarningMemoryUtilization
- vSphereDatastoreWarningDiskUtilization
- vSphereDatastoreCriticalDiskUtilization
- vSphereHostWarningHighPacketErrors

> ## vSphere Cluster

Reviewer: This dashboard is called `vSphere clusters` on the spec.

> The vSphere cluster dashboard provides details on cluster CPU and memory while giving a high level view into hosts, and VM's.

Reviewer: ```suggestion
The vSphere cluster dashboard provides details on cluster CPU and memory while giving a high-level view of hosts and VMs.
```

> ![vSphere Overview Dashboard]()

Reviewer: ```suggestion
![vSphere cluster dashboard]()
```

Reviewer: I'm suggesting changing the capitalization to sentence case to match how the dashboard is named in Grafana.

## vSphere Overview

The vSphere overview dashboard provides details on CPU, memory, resource pools, ESXi hosts, and datastores.

![vSphere Overview Dashboard]()

> 

Reviewer: Missing sections for the `hosts` and `virtual machines` dashboards.
